miso-web/src/main/webapp/scripts/task_ajax.js

```diff
@@ -300,11 +242,12 @@ Tasks.ui = {
       for (var j = 0; j < pipeline.processes.length; j++) {
         var process = pipeline.processes[j];
         for (var k = 0; k < process.parameters.length; k++) {
-          if (jQuery('input[name="'+process.parameters[k].name+'"]').length === 0) {
-            jQuery('#'+pipeline.name+'-reqParams > tbody:last')
-              .append(jQuery('<tr>')
-              .append("<td>"+process.parameters[k].name+"</td><td><input style='width:98%' optional='true' type='text' id='"+process.parameters[k].name+"' name='"+process.parameters[k].name+"' value=''/>")
-            );
+          if (jQuery('input[name="' + process.parameters[k].name + '"]').length == 0) {
+            jQuery('#' + pipeline.name + '-reqParams > tbody:last').append(
+                jQuery('<tr>').append("<td>" + process.parameters[k].name + "</td>").append(
+                    "<td><input style='width:98%' optional='true' type='text'").append(
```

Comment:
Adding an `append` here will cause jQuery to close the `input` element before appending, so you'll end up with an input box followed by text of `id='name' name='name'...`.

Is there a particular reason for using `append`? It's is over twice as slow as `+`...